src/index.tsx

```diff
@@ -91,7 +98,7 @@ function App() {
   });
 
   const [mapId] = useState(uuidv4());
-
+  let mapRef = useRef<DeckGLRef>(null);
```

Comment:
nit: this and the useState above can all be const

Reply:
I think this is me writing too much Rust and always using `let`...